src/aliases/store/postgres/alias.go

```diff
@@ -0,0 +1,58 @@
+package aliaspg
+
+import (
+	"context"
+	goerrors "errors"
+
+	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+	models "github.com/consensys/quorum-key-manager/src/aliases/store/models"
+	"github.com/consensys/quorum-key-manager/src/infra/postgres"
+)
+
+var _ aliasstore.Alias = &Alias{}
+
+// Alias stores the alias data in a postgres DB.
+type Alias struct {
+	pgClient postgres.Client
+}
+
+func NewAlias(pgClient postgres.Client) *Alias {
+	return &Alias{
+		pgClient: pgClient,
+	}
+}
+
+func (s *Alias) CreateAlias(ctx context.Context, registryName models.RegistryName, alias models.Alias) error {
```

Comment:
You are missing the returned object in the create function
```
func (ea *ETH1Accounts) Add(ctx context.Context, account *entities.ETH1Account) (*entities.ETH1Account, error) {
```

Reply:
I'm ok with it.